packages/projects/src/components/ProjectForm.jsx

```diff
@@ -80,12 +83,12 @@ const reducer = (state, action) => {
 
 const formClass = 'mt-1 block w-full rounded-md border-gray-300 shadow-sm focus:border-indigo-300 focus:ring focus:ring-indigo-200 focus:ring-opacity-50'
 
-const change = (dispatch, type, e) => {
+const change = (state, dispatch, type, e) => {
   dispatch({ type: 'formStatus', payload: FORM_STATUSES.clean })
 
   try {
```

Comment:
Why are we wrapping this in a try/catch block here?

Reply:
I don't know. You wrote it.
https://github.com/kernel-community/services/commit/f7054dd1b1565d45788be8c0cb38262a884213ea#diff-c05337e069f8e4474a51c9ec9eab07a3f23868ffc52ac5024c6dc58ca654bcecR61